features/premium/subscribe_self_to_premium.feature

```diff
@@ -6,15 +6,25 @@ Feature: Subscribe Self to Premium
 
   Background:
     Given the following plans exist
-      | name         | id          | amount | free_trial_length_days |
-      | Premium      | premium     | 1000   | 7                      |
+      | name                       | id                       | amount | free_trial_length_days |
+      | Premium                    | premium                  | 1000   | 7                      |
+      | Premium (first month free) | premium_first_month_free | 1000   | 31                     |
     And the email queue is clear
 
   Scenario: Pay by card
     Given I visit "subscriptions/new"
     And I click "Subscribe" within the card_section
     When I fill in appropriate card details for premium
     Then I should see "Thanks, you're now an AgileVentures Premium Member!"
+    And I should see "Your 7 day free trial has now started"
+    And "random@morerandom.com" should receive a "Welcome to AgileVentures Premium" email
+
+  Scenario: Pay by card (with free trial month)
+    Given I visit "subscriptions/new?plan=av_premium_first_month_free"
+    And I click "Subscribe" within the card_section
+    When I fill in appropriate card details for premium
+    Then I should see "Thanks, you're now an AgileVentures Premium Member!"
```

Comment:
shouldnt this say 31 day free trial? @tansaku

Reply:
yes, just fixing that

Reply:
just checking it fails locally if I remove the fix